pilot/pkg/features/experimental.go

```diff
@@ -25,6 +25,10 @@ import (
 
 // Define experimental features here.
 var (
+	EnablePassthroughConnections = env.Register("PILOT_ENABLE_PASSTHROUGH_CONNECTIONS", true,
+		"If enabled, HTTP pooling will be disabled on the passthrough cluster. This means each downstream connection will result in one upstream connection. " +
```

Comment:
Not sure why http pool here as it applies to PassThrough cluster

Reply:
PassthroughCluster can do http pooling if we already sent it through the HCM. See the examples in the PR description and envoy issue I linked that explain the problems that creates